watertap/data/techno_economic/zeroth_order_model_data/aeration_basin.yaml

```diff
@@ -0,0 +1,41 @@
+default:
+  default_removal_mass_solute:
+    unit: kg/kg
+    value: 0
+  energy_electric_flow_vol_inlet:
+    unit: kWh/m^3
+    value: 0.41226735
+  recovery_vol:
+    reference: .nan
```

Comment:
Minor comment, but for a reference I would personally have used None (or omitted the reference line) rather than `.nan`. This is probably just semantics however (and this entry is just for information and not used currently).

Reply:
Oh good point. I did not notice it was doing that. I will update that to None (Think this is a quirk from csv import)